web/templates/typeahead_list_item.hbs

```diff
@@ -35,12 +35,16 @@
     {{> status_emoji status_emoji_info}}
     {{~/if}}
     {{~#if has_secondary}}
+    {{#if has_rendered_markdown_secondary}}
+    <small class="autocomplete_secondary rendered_markdown">{{{secondary}}}</small>
+    {{else}}
     <small class="autocomplete_secondary">
         {{~ secondary ~}}
     </small>
-    {{#if is_unsubscribed}}
-    <span class="fa fa-exclamation-triangle unsubscribed_icon"
-      title="{{t 'You are not currently subscribed to this channel.' }}"></span>
     {{/if}}
     {{~/if}}
 </div>
+{{#if is_unsubscribed}}
+    <span class="fa fa-exclamation-triangle unsubscribed_icon"
+      title="{{t 'You are not currently subscribed to this channel.' }}"></span>
+{{/if}}
```

Comment:
Why is this pulled out of the `div`? That change should be its own commit with its own commit message explanation if you feel the need to do it.

Reply:
The icon appeared on the next line when it was inside `div`. So I moved it out. Added a separate commit for this.